<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="environmentalImpactProgressBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
border-color: white;
border-radius: 5px;
border-width: 2px;
	color: white;

 }

 QProgressBar::chunk {
     background-color: rgb(89, 163, 72);
 }
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="foodProgressBar">
       <property name="styleSheet">
        <string notr="true">QProgressBar {
border-color: white;
border-radius: 5px;
border-width: 2px;
	color: white;

 }

 QProgressBar::chunk {
     background-color: rgb(89, 163, 72);
 }
</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="funProgressBar">
       <property name="styleSheet">
        <string notr="true">QProgressBar {
border-color: white;
border-radius: 5px;
border-width: 2px;
	color: white;

 }

 QProgressBar::chunk {
     background-color: rgb(89, 163, 72);
 }
</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="amenitiesProgressBar">
       <property name="styleSheet">
        <string notr="true">QProgressBar {
border-color: white;
border-radius: 5px;
border-width: 2px;
	color: white;

 }

 QProgressBar::chunk {
     background-color: rgb(89, 163, 72);
 }
</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="energyProgressBar">
       <property name="styleSheet">
        <string notr="true">QProgressBar {
border-color: white;
border-radius: 5px;
border-width: 2px;
	color: white;

 }

 QProgressBar::chunk {
     background-color: rgb(89, 163, 72);
 }
</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Environmental &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Impact:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Fun:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Food:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Amenities:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Energy:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListView" name="cleanEnergyListView">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>256</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="dirtyEnergyListView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>256</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
